Gordon360/Controllers/MembershipsController.cs

```diff
@@ -83,6 +83,21 @@ public ActionResult<int> GetMembershipCount(string? activityCode = null, string?
             return Ok(result);
         }
 
+        /// <summary>
+        /// Gets the memberships of the given user
+        /// </summary>
+        /// <param name="username"> username filter </param>
+        /// <returns>The number of followers of the activity</returns>
```

Comment:
If we don't delete this endpoint, then the comment will need improving.  The returns explanation looks wrong, and the summary should be more explicit about what's included: all memberships over all terms?  just public?  just those visible to the current user (so different for own profile than public profiles)?